src/app/clusters/sample-mei-server/sample-mei-server.cpp

```diff
@@ -60,6 +60,7 @@ SampleMeiContent::SampleMeiContent() : SampleMeiContent(kInvalidEndpointId) {}
 SampleMeiContent::SampleMeiContent(EndpointId aEndpoint)
 {
     endpoint = aEndpoint;
+    pingCount = 10000;
```

Comment:
Any reason to set the pingCount starts from 10000?

Reply:
We could set it to zero. Just wanted to see non-default data from the start.